packages/babel-cli/src/babel/util.js

```diff
@@ -6,6 +7,7 @@ let util      = require("babel-core").util;
 let path      = require("path");
 let fs        = require("fs");
 let _         = require("lodash");
+let each      = require("lodash/each");
```

Comment:
It was to keep it consistent with other modules where each has been used like this, but I have changed it as per your suggestion.